kafka/consumer/group.py

```diff
@@ -128,6 +128,9 @@ class KafkaConsumer(six.Iterator):
             offset commits; 0.8.0 is what is left. If set to 'auto', will
             attempt to infer the broker version by probing various APIs.
             Default: auto
+        client_check_version_timeout_s (int): number of seconds to throw a
```

Comment:
I know, the timeout argument already written in to client.check_version accepts seconds.

it seems ugly to convert so if we want ms then I'll change client.check_version to use ms